### PR TITLE
Remove Enzyme As A Dependency - RSC-1424

### PR DESCRIPTION
**Pull request by avnvbhatta**
https://issues.sonatype.org/browse/RSC-1424

### CODE REVIEW

File: lib/src/__testutils__/rtlUtils.tsx [hunk 1]
```diff
@@ -7,7 +7,6 @@
 import React, { ComponentType } from 'react';
 import { act, render, RenderResult } from '@testing-library/react';
 
-import '@testing-library/jest-dom';
```
Comment (rpokorny) [inline lib/src/__testutils__/rtlUtils.tsx:10]: This isn't still necessary?

Comment (rpokorny) [reply lib/src/__testutils__/rtlUtils.tsx:10]: > This isn't still necessary?

Nvm I see you added it in `setupTests`

File: gallery/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "12.10.21",
```
Comment (rpokorny) [inline gallery/package.json:3]: Version bumps not expected; this is being merged to the feature branch

**Review by rpokorny — APPROVED** [review]
+1 aside from the version bumps

File: gallery/package.json (hunk 2 above)
Comment (avnvbhatta) [reply gallery/package.json]: > Version bumps not expected; this is being merged to the feature branch

It was only to trigger CI. Reverted here: c078212807fd8ba661742406d1282efbed3d16d5

**Pull request merged**